FeedStoreChallenge/CoreDataFeedStore.swift

```diff
@@ -4,7 +4,7 @@
 
 import CoreData
 
-public final class CoreDataFeedStore: FeedStore {
+final class CoreDataFeedStore: FeedStore {
```

Comment:
This class should be `public` to be visible from outside of the module. Otherwise it would not be available for usage by module's clients.

Please make it `public`.
```suggestion
public final class CoreDataFeedStore: FeedStore {
```